book/B-embedding-git/sections/libgit2.asc

```diff
@@ -133,7 +133,7 @@ error = git_repository_open(&repo, "some-path");
 error = git_repository_set_odb(repo, odb); // <4>
 ----
 
-_(Note that errors are captured, but not handled. We hope your code is better than ours.)_
+_Note that errors are captured, but not handled. We hope your code is better than ours._
```

Comment:
I'm a _tiny_ bit sad to see this go. The parens add something. _sigh_ I guess it's the right thing though.